modules/users/client/components/ProfileTabs.component.js

```diff
@@ -0,0 +1,103 @@
+// External dependencies
+import { useTranslation } from 'react-i18next';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+
+// Internal dependencies
+import { $on } from '@/modules/core/client/services/angular-compat';
+import { getCount as getExperiencesCount } from '@/modules/references/client/api/references.api';
+
+export default function ProfileTabs({
+  contactsCount,
+  initialPathName,
+  isExperiencesEnabled,
+  isOWnProfile,
+  userId,
+  username,
+}) {
+  const { t } = useTranslation('users');
+  const [experiencesCount, setExperiencesCount] = useState(0);
+  const [pathName, setPathName] = useState(initialPathName);
+
+  useEffect(async () => {
+    const count = await getExperiencesCount(userId);
```

Comment:
if experiences are disabled, the call to back-end will still be made, right?

Reply:
Good catch! Fixed in https://github.com/Trustroots/trustroots/pull/1970/commits/8fbda57ab677e83472bbe87be2796a75b183b652